frontend/src/components/Header.tsx

```diff
@@ -24,6 +25,20 @@ type Props = {
   locale?: string;
 };
 
+const NAV_PATHS = ["/search", "/process", "/research", "/subscribe"];
```

Comment:
Might be cleaner to use the `navLinkList()` list so we don't have to declare the paths twice.